electron/main.js

```diff
@@ -254,6 +254,8 @@ let sortMethod = settings.get('sortMethod', 'draw');
 let useFlat = settings.get('useFlat', true);
 let useMinimal = settings.get('useMinimal', true);
 let zoom = settings.get('zoom', 0.8);
+let recentCards = settings.get('recentCards', []);
```

Comment:
OK, good, this is using electron-settings. There isn't anything in [the docs]() warning against heavy use; only that settings shouldn't be used from the renderer process and only the main process. It looks like we pass [that check](https://github.com/mtgatracker/mtgatracker/search?q=electron-settings&unscoped_q=electron-settings)